pallets/offchain-worker/src/urls.rs

```diff
@@ -96,8 +96,8 @@ pub enum HttpRequest<'a> {
 }
 
 /// Store all API tokens for offchain worker to send request to website
-#[serde(crate = "alt_serde")]
 #[derive(Deserialize, Encode, Decode, Default)]
+#[serde(crate = "alt_serde")]
```

Comment:
Why exchange the place? Is there a norm for this?

Reply:
It is from compiler's suggestion, then avoid the warning.